### DISCUSSION TITLE
How does the tracker blocker work ?

### DISCUSSION THREAD

**Discussion by edern-appere** [Divested-Mobile/DivestOS-Build#106] [General] [answered]
Hi there,

I'm trying to learn more about DivestOS before eventually making the switch. I've read that there is a tracker blocker built-in. How does it work ? If it blocks some domain that an app requires, is it possible to unblock ?
Right now I'm running LOS+microg + I have installed Blokada. If one app cannot run because of one domain blocked, I can ask Blokada to make an exception for this app. 
Also, if I switch to DivestOS, is it still usefull to have Blokada installed ?

Thanks for your help !

**Comment by SkewedZeppelin** [comment, marked as answer]
The tracker blocker is a hardcoded list of domains to block.
The lists are here: https://divested.dev/index.php?page=dnsbl
DivestOS uses the hosts-wildcards format.
You can click it to see all the sources it combines.

18.1 and 19.1 have an option in Settings to disable the list.
But there is no way to disable it per-app.

There is a preemptive allowlist to prevent breaking many common websites/apps.
If you find any false positives please report to me and I'll exclude them:
https://github.com/divestedcg/dnsrm

**Comment by edern-appere** [comment]
Thanks for your answer. When I'll install the ROM I will try and let you know if I encounter any issue.

If I understand well, using your tracker blocker, I do not need Blokada anymore (since they do the same thing) ?

**Comment by SkewedZeppelin** [comment]
The tracker blocker is a hardcoded list of domains to block.
The lists are here: https://divested.dev/index.php?page=dnsbl
DivestOS uses the hosts-wildcards format.
You can click it to see all the sources it combines.

18.1 and 19.1 have an option in Settings to disable the list.
But there is no way to disable it per-app.

There is a preemptive allowlist to prevent breaking many common websites/apps.
If you find any false positives please report to me and I'll exclude them:
https://github.com/divestedcg/dnsrm
